app/bundles/UserBundle/Security/Authenticator/Passport/Badge/PluginBadge.php

```diff
@@ -31,6 +31,6 @@ public function getAuthenticatingService(): ?string
 
     public function isResolved(): bool
     {
-        return null !== $this->preAuthenticatedToken || null !== $this->pluginResponse;
+        return true;
```

Comment:
This needs some sight. When authentication is successful this is always going to be both `null` at the moment if you're authentication doesn't use a custom token (i.e. calls `setIsAuthenticated` on the event instead of `setToken`